### PR TITLE
Remove deprecated functions

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Removed the deprecated functions and updated the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -62,6 +62,22 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
 
 ### Deprecated
```
Comment (vuvoth) [inline CHANGELOG.md:63]: I think it should in breaking changed

**Review by vuvoth — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -62,6 +62,22 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
...
 
+In `deployer` API:
```
Comment (ratik21) [inline CHANGELOG.md:65]:
```suggestion
#### @algo-builder/algob
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -62,6 +62,22 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
...
+
+- `ensureCompiled` is removed and `compileASC` should be used.
+- `loadLogicFromCache` is removed and `getLsigFromCache` should be used.
+- `executeTransaction` is removed and `executeTx` should be used.
+
+In `runtime` API:
```
Comment (ratik21) [inline CHANGELOG.md:71]: same as above

File: CHANGELOG.md [hunk 4]
```diff
@@ -62,6 +62,22 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
 
 ### Deprecated
 
+In `deployer` API:
+
+- `ensureCompiled` is removed and `compileASC` should be used.
+- `loadLogicFromCache` is removed and `getLsigFromCache` should be used.
+- `executeTransaction` is removed and `executeTx` should be used.
```
Comment (ratik21) [inline CHANGELOG.md:69]: this is api breaking

File: CHANGELOG.md (hunk 1 above)
Comment (Megha-Dev-19) [reply CHANGELOG.md]: updated

**Pull request merged**